source/guides/api-key-management/index.markdown

```diff
@@ -195,7 +195,7 @@ Stormpath supports two HTTP `Authorization` methods when authenticating API Keys
 ### How API Key and Token Authentication Works
 All authentication attempts in Stormpath start with the `Application` object in the SDK.  You will likely have initialized the `Application` during startup.
 
-The `Application` object in Java has an `authenticateApiRequest` method that takes an HTTP request as a parameter.  Using  HTTP authorization headers, Stormpath can understand what type of authentication is occurring (Basic vs Bearer) and can quickly decide if the authentication request is successful or not.  For example, if your API supports both Basic and OAuth 2.0, the Stormpath SDK will take the full request, read the authorization header data and provide the right type of authentication.
+Then you will need to get an instance of `ApiRequestAuthenticator` to authenticate, you can get this using `Applications` and pass the application as parameter to the `apiRequestAuthenticator` method `Applications.apiRequestAuthenticator(application)` once you have this you can authenticate using the HTTP request as a parameter.  Using  HTTP authorization headers, Stormpath can understand what type of authentication is occurring (Basic vs Bearer) and can quickly decide if the authentication request is successful or not.  For example, if your API supports both Basic and OAuth 2.0, the Stormpath SDK will take the full request, read the authorization header data and provide the right type of authentication.</p>
```

Comment:
"... you can obtain it using the `Applications` utility class while passing the `Application` object as a parameter like this: `Applications.oauthRequestAuthenticator(application)`. You can then execute the actual authentication attempt using the HTTP request as a parameter by means of its `authenticate(HttpRequest)` method."